client/src/components/Dashboard-FormAssemply.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import * as S from './Dashboard-FormAssemply.style';
+import * as C from './Journal.style';
```

Comment:
I get that you want to share styles, but i think that maybe it might be better for components to have their own styles instead of sharing.